### ISSUE TITLE
Loosing image quality

### ISSUE THREAD

**Issue by jigneshvashundhara** [Zomato/AndroidPhotoFilters#42]
After applying filter on image image loss quality even i choose HD images

**Comment by Ramprasadgit** [comment]
Yes. The resolution is reducing to too much from 2.50 MB quality of image to 130 KB around.
